api.jsx

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const ncNewsAPI = axios.create({
+  baseURL: `https://northcoders-news-ykvv.onrender.com/api`,
+});
+
+export const fetchArticles = async (topic) => {
+  try {
+    const response = await ncNewsAPI.get(`/articles?topic=${topic}`);
```

Comment:
This ticket was only meant to be to bring all the articles, without any option for filtering by topic at this point. You've actually done Ticket 10 as well here!

Reply:
Whoops, apologies! I must look ahead too...